template/org.html

```diff
@@ -109,8 +109,9 @@ <h1><a href="/github.com/{{ .Repo }}@{{ .Tag }}">{{ .Repo }}@{{ .Tag }}</a></h1>
             </table>
         </div>`;
     }
-
+    //
```

Comment:
I needed to add these so the rendering of the template worked... very weird stuff. I debugged this for like 5 hours.